### PR TITLE
FIX Reorder model tabs

### PR DESCRIPTION
**Pull request by emteknetnz**
Fixes https://github.com/silverstripe/silverstripe-versioned/issues/207

You can get the Files tab to easily show by return true from `FileArchiveExtension::isArchiveFieldEnabled()`

(most installations will probably have this set to false and won't show the Files tab)

Default tab is selected

Before
image

After
image

### CODE REVIEW

**Comment by emteknetnz** [comment]
image

File: lang/en.yml [hunk 1]
```diff
@@ -7,6 +7,6 @@ en:
     COLUMN_TYPE: Type
     SELECT_EMPTY: Select…
     SELECT_TYPE: 'Select a content type'
-    TAB_OTHERS: Others
+    TAB_OTHERS: Other
```
Comment (robbieaverill) [inline lang/en.yml:10]: Can you create a new key? `TAB_OTHER` seems like a nice name

File: src/ArchiveAdmin.php [hunk 2]
```diff
@@ -324,6 +331,13 @@ public function getManagedModelTabs()
         $forms = ArrayList::create();
         $mainModels = $this->getVersionedModels('main', true);
 
+        // Display order should be Pages > Blocks > Files > Other
+        // Pages, Blocks, Files are treated specially and have extensions defined in _config/archive-admin.yml
```
Comment (robbieaverill) [inline src/ArchiveAdmin.php:335]: Where do user defined tabs come into this? (Aside from "other")

File: lang/en.yml (hunk 1 above)
Comment (emteknetnz) [reply lang/en.yml:10]: But we're changing an existing label to something else, seems like not updating the key is the correct thing.  Also it'll mean that any other languages will loose their translation.  I think we should keep the slightly weird key as is.

File: src/ArchiveAdmin.php (hunk 2 above)
Comment (emteknetnz) [reply src/ArchiveAdmin.php:335]: I'm not sure, I presume at either before or after the 3 special tabs.  I'm assuming that they add tabs via an extension and yml similar to https://github.com/silverstripe/silverstripe-versioned-admin/blob/1/_config/archive-admin.yml.  Probably after these 3 tabs though before 'Other'?  I would expect the uasort() function to not impact the user defined tabs.

**Review by bergice — APPROVED** [review]

**Pull request merged**